common/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "common",
+  "version": "0.0.1-prealpha",
+  "main": "index.ts",
+  "private": true,
+  "dependencies": {
+    "adm-zip": "^0.4.13",
```

Comment:
How'd you pick adm-zip?

I think we may want to use jszip. It's the library used in scratch-vm. I need to look more to compare them but jszip looks to have more support.

Reply:
Cool! If jszip is preferred, I'll definitely check it out, and use it. I chose adm-zip because of its popularity

Reply:
+1 on this, I would use the same lib as scratch, and ideally the same version so we can have flattened dependencies if/when possible.

Reply:
Replaced adm-zip with jszip